conan/tools/files/files.py

```diff
@@ -263,6 +264,47 @@ def chdir(conanfile, newdir):
         os.chdir(old_path)
 
 
+def chmod(conanfile, path:str, read:bool, write:bool, execute:bool, recursive:bool=False):
+    """
+    Change the permissions of a file or directory. The same as the Unix command chmod, but simplified.
+    It skips in case running on Windows.
+
+    :param conanfile: The current recipe object. Always use ``self``.
+    :param path: Path to the file or directory to change the permissions.
+    :param read: If ``True``, the file or directory will have read permissions for any user.
+    :param write: If ``True``, the file or directory will have write permissions for any user.
+    :param execute: If ``True``, the file or directory will have execute permissions for any user.
+    :param recursive: (Optional, Defaulted to ``False``) If ``True``, the permissions will be applied
+    """
+    if platform.system() == "Windows":
```

Comment:
Why? ConanCenter recipes very rarely do this, only in a couple of occcasions.
And chmod in Windows still can change the read-only flag, so if the function has ``read`` argument, then it should work.

Reply:
Okay, but what should be done when passing execute or write? Just ignore or rise an error?